pkg/scanner/local/scan.go

```diff
@@ -183,14 +196,14 @@ func (s Scanner) scanLibrary(apps []ftypes.Application, options types.ScanOption
 
 func skipped(filePath string, skipFiles, skipDirectories []string) bool {
 	for _, skipFile := range skipFiles {
-		skipFile = strings.TrimLeft(filepath.Clean(skipFile), string(os.PathSeparator))
+		skipFile = strings.TrimLeft(skipFile, string(os.PathSeparator))
```

Comment:
This `filepath.Clean` is necessary.

Reply:
Already removed, don't know why GitHub shows it